### PR TITLE
Quieter logs

### PR DESCRIPTION
**Pull request by tylerkaraszewski**
### Fixed Issues
Fixes GH_LINK

### Tests

_________
**Internal Testing Reminder:** when changing bedrock, please compile auth against your new changes

### CODE REVIEW

**Review by justinpersaud — APPROVED** [review]

File: BedrockCore.cpp [hunk 1]
```diff
@@ -171,7 +171,6 @@ BedrockCore::RESULT BedrockCore::peekCommand(unique_ptr<BedrockCommand>& command
         response["commitCount"] = to_string(_db.getCommitCount());
 
         // Success. If a command has set "content", encode it in the response.
-        SINFO("Responding '" << response.methodLine << "' to read-only '" << request.methodLine << "'.");
```
Comment (iwiznia) [inline BedrockCore.cpp:174]: This is kind of useful, PHP already logs it but we would not have this for all-auth processes though (like billing and such)...

**Review by iwiznia — APPROVED** [review]
Have my doubts on the one I commented, though being fair I don't think I ever needed this for non PHP initiated commands, so approving

File: BedrockCore.cpp (hunk 1 above)
Comment (tylerkaraszewski) [reply BedrockCore.cpp:174]: We also get a timing log when each command completes. This is only differently useful if we're particularly looking at whether or not we sent a response.

Comment (iwiznia) [reply BedrockCore.cpp:174]: But that does not log the response code... though maybe it does not matter as there should always be an exception if we don't return 200?
Anyway, I already approved 😄

**Pull request merged**